src/main/scala/it/pagopa/interop/partyregistryproxy/service/impl/package.scala

```diff
@@ -54,17 +53,6 @@ package object impl {
     }
   }
 
-  def useWriter[A](writer: Try[IndexWriter], f: IndexWriter => Try[A], zero: A): Try[A] = writer match {
-    case Success(wr)                           => f(wr)
-    case Failure(_: LockObtainFailedException) => Success(zero)
```

Comment:
I did not include this case yet, is it ok to ignore an error on lock e silently skip update?

Reply:
I won't expect that the rate of the changes is so strict to require a sophisticated error handling